gcp_variant_transforms/testing/integration/run_preprocessor_tests.py

```diff
@@ -87,40 +97,57 @@ def validate_result(self):
     """Validates the results.
 
     - Checks that the report is generated.
-    - Validates report's contents are the same as `expected_contents`.
+    - Validates report's contents contain the strings in `expected_headlines`.
+    - Checks that the resolved headers are generated if `header_blob_name` is
+    specified in the test.
     """
     client = storage.Client(self._project)
     bucket = client.get_bucket(_BUCKET_NAME)
-    blob = bucket.get_blob(self._blob_name)
-    if not blob.exists(client):
+    report_blob = bucket.get_blob(self._report_blob_name)
+    if not report_blob:
       raise run_tests_common.TestCaseFailure(
           'Report is not generated in {}'.format(self._report_path))
-    contents = blob.download_as_string()
+    contents = report_blob.download_as_string()
     expected_contents = ''.join(self._expected_contents)
```

Comment:
I think it's easier to do `'\n'.join(self._expected_contents)` to avoid adding `\n` to every line in the test cases. WDYT?